example/tsconfig.json

```diff
@@ -0,0 +1,62 @@
+{
```

Comment:
I believe I have configured the example to use the tsconfig from the root folder, is there a specific reason you added a second one specifically for the example?